src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
move random to the class constant, like: 
![image](https://user-images.githubusercontent.com/12864128/221845920-190e4f8a-0749-47bc-811a-c2075d152a07.png)